src/lib/pickles/wrap.mli

```diff
@@ -1,162 +1,132 @@
-         , Impls.Wrap.Impl.field Snarky_backendless.Cvar.t
-         , Impls.Wrap.Impl.field Snarky_backendless.Cvar.t
-         , Impls.Wrap.Impl.field Snarky_backendless.Cvar.t
-         , ( Impls.Wrap.Impl.field Snarky_backendless.Cvar.t
-             Kimchi_backend_common.Scalar_challenge.t
-             Composition_types.Bulletproof_challenge.t
-           , Pickles_types.Nat.z Backend.Tick.Rounds.plus_n )
-           Pickles_types.Vector.t
-           Pickles_types.Hlist0.Id.t
-         , Impls.Wrap.Impl.field Snarky_backendless.Cvar.t )
-         Import.Types.Wrap.Statement.In_circuit.t
+         , Pasta_bindings.Fq.t Challenge.t
+         , Pasta_bindings.Fq.t Challenge.t
+         , Pasta_bindings.Fq.t Challenge.t
+         , ( Pasta_bindings.Fq.t Challenge.t Kimchi_types.scalar_challenge
+             Bulletproof_challenge.t
+           , Nat.z Backend.Tick.Rounds.plus_n )
+           Vector.vec
+         , Pasta_bindings.Fq.t Challenge.t )
+         Types.Wrap.Statement.In_circuit.t
       -> unit )
   -> typ:('a, 'b) Impls.Step.Typ.t
   -> step_vk:Kimchi_bindings.Protocol.VerifierIndex.Fp.t
-  -> actual_wrap_domains:(Core_kernel.Int.t, 'c) Pickles_types.Vector.t
+  -> actual_wrap_domains:(int, 'c) Vector.vec
   -> step_plonk_indices:'d
   -> actual_feature_flags:bool Plonk_types.Features.t
   -> ?tweak_statement:
-       (   ( Import.Challenge.Constant.t
-           , Import.Challenge.Constant.t Import.Types.Scalar_challenge.t
-           , Backend.Tick.Field.t Pickles_types.Shifted_value.Type1.t
-           , ( Backend.Tick.Field.t Pickles_types.Shifted_value.Type1.t
-             , bool )
-             Import.Types.Opt.t
-           , ( Import.Challenge.Constant.t Import.Types.Scalar_challenge.t
-               Composition_types.Wrap.Proof_state.Deferred_values.Plonk
-               .In_circuit
-               .Lookup
-               .t
+       (   ( Challenge.Constant.t
+           , scalar_challenge_constant
+           , Pasta_bindings.Fp.t Shifted_value.Type1.t
+           , ( scalar_challenge_constant
+               Types.Wrap.Proof_state.Deferred_values.Plonk.In_circuit.Lookup.t
              , bool )
-             Import.Types.Opt.t
+             Plonk_types.Opt.t
            , bool
            , 'max_proofs_verified
-             Proof.Base.Messages_for_next_proof_over_same_field.Wrap.t
-           , (int64, Composition_types.Digest.Limbs.n) Pickles_types.Vector.vec
+             Reduced_messages_for_next_proof_over_same_field.Wrap.t
+           , (int64, Nat.N4.n) Vector.vec
            , ( 'b
-             , ( Kimchi_pasta.Pallas_based_plonk.Proof.G.Affine.Stable.V1.t
+             , ( Backend.Tock.Proof.G.Affine.t
                , 'actual_proofs_verified )
-               Pickles_types.Vector.t
-             , ( ( Import.Challenge.Constant.t Import.Scalar_challenge.t
-                   Import.Bulletproof_challenge.t
+               Vector.vec
+             , ( ( Challenge.Constant.t Kimchi_types.scalar_challenge
```

Comment:
Why this change?

Reply:
Same, can we do that in https://github.com/MinaProtocol/mina/pull/14081?